lib/widgets/pay_date_dropdown.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import '../theme/theme.dart';
+import 'string_constants.dart';
+
+class PayDateDropdown extends StatefulWidget {
+  @override
+  State<PayDateDropdown> createState() => _PayDateDropDownState();
+}
+
+class _PayDateDropDownState extends State<PayDateDropdown> {
+  DateTime? selectedDate;
+
+  Future<void> _selectDate(BuildContext context) async {
+    final picked = await showDatePicker(
+      context: context,
+      initialDate: selectedDate ?? DateTime.now(),
+      firstDate: DateTime(2000),
+      lastDate: DateTime(2101),
```

Comment:
use lastdate as today, as we should not add expense for future !! lastDate: DateTime.now(),

Reply:
done